src/libraries/System.Drawing.Common/src/Interop/Windows/Interop.Comdlg32.cs

```diff
@@ -11,14 +11,14 @@ internal static partial class Comdlg32
 #pragma warning disable DLLIMPORTGENANALYZER015 // Use 'GeneratedDllImportAttribute' instead of 'DllImportAttribute' to generate P/Invoke marshalling code at compile time
         // TODO: [DllImportGenerator] Switch to use GeneratedDllImport once we support non-blittable structs.
```

Comment:
Not that you have changed `PRINTDLG` to blittable struct, you can delete this TODO and switch the PInvokes to use `GeneratedDllImport`